src/rdf4cpp/rdf/Literal.cpp

```diff
@@ -310,6 +347,10 @@ Literal Literal::as_simplified_lexical_form(NodeStorage &node_storage) const noe
 
 std::string_view Literal::language_tag() const noexcept {
     if (this->datatype_eq<datatypes::rdf::LangString>()) {
+        if (this->is_inlined()) {
+            auto id = rdf4cpp::rdf::datatypes::registry::lang_tags::from_inlined(this->handle_.node_id().literal_id().value);
```

Comment:
Not really important more fyi if you didn't know yet. C++ now (since 17) has [structured bindings](https://en.cppreference.com/w/cpp/language/structured_binding), which can destructure some things into their elements. For example this line could be rewritten as

```c++
auto [tag, _] = rdf4cpp::rdf::datatypes::registry::lang_tags::from_inlined(this->handle_.node_id().literal_id().value);
```